lib/array_intersection.rb

```diff
@@ -1,6 +1,27 @@
 # Returns a new array to that contains elements in the intersection of the two input arrays
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n*m) because there is a nested loop, having to go through each element in both arrays
+# Space complexity: O(n) where n is the number of elements in the intersection array
```

Comment:
This space complexity would be better described as O(min(n, m)) where n and m are the lengths of the two input arrays.